pkg/cmd/pipeline/translate.go

```diff
@@ -136,6 +136,27 @@ func UpdateConfigMap(ctx context.Context, cmap *apiv1.ConfigMap, data *CfgData,
 	return configmaps.Deploy(ctx, cmap, cmap.Namespace, c)
 }
 
+// UpdateEnvs updates the configmap adding the envs as data fields
+func UpdateEnvs(ctx context.Context, name, namespace string, envs []string, c kubernetes.Interface) error {
+	cmap, err := configmaps.Get(ctx, TranslatePipelineName(name), namespace, c)
+	if err != nil {
+		return err
+	}
+
+	if cmap != nil {
+		for _, env := range envs {
+			result := strings.Split(env, "=")
+			if len(result) != 2 {
+				return fmt.Errorf("invalid env format: '%s'", env)
+			}
+
+			cmap.Data[fmt.Sprintf("%s_%s", constants.OktetoDependencyEnvPreffix, result[0])] = result[1]
```

Comment:
we need to add a prefix in order to know which values in the config map should be retrieved and set it as an environment variable